server/src/main/java/org/apache/iotdb/db/service/thrift/impl/TSServiceImpl.java

```diff
@@ -170,21 +93,12 @@
   private static final String INFO_INTERRUPT_ERROR =
       "Current Thread interrupted when dealing with request {}";
 
-  private static final int MAX_SIZE = CONFIG.getQueryCacheSizeInMetric();
-  private static final int DELETE_SIZE = 20;
-
-  private static final List<SqlArgument> sqlArgumentList = new ArrayList<>(MAX_SIZE);
-
   private long startTime = -1L;
```

Comment:
```suggestion
```

Can the startTime be deleted?

Reply:
Let me take a look

Reply:
This parameters may used by tracing, so I can delete this.

Reply:
Ignore it :D